fusesoc/coremanager.py

```diff
@@ -161,6 +171,17 @@ def eq_vln(this, that):
         cores = [x["core"] for x in self._cores.values()]
         conflict_map = self._get_conflict_map()
 
+        # Enumerate cores in lock file, if any
+        lockfile_virtuals = {}
```

Comment:
Should lockfile_virtuals still be here after we stripped out the virtuals support from this PR?

Reply:
No, should not be in there,